### DISCUSSION TITLE
mitmproxy2swagger does not progress

### DISCUSSION THREAD

**Discussion by dankpanda0** [mitmproxy/mitmproxy#6783] [General]
Hi,

I am using mitmproxy2swagger in kali and it gets stuck here - 

![image](https://github.com/mitmproxy/mitmproxy/assets/92813350/7c10ff25-d79a-4c1b-a5ae-e10ea6887395)

I have installed it using pip and all the packages have been updated especially pyopenssl, based on similar discussions.

**Comment by Prinzhorn** [comment]
This is the wrong repo, I assume you're using https://github.com/alufers/mitmproxy2swagger

**Comment by dankpanda0** [comment]
Ohh right, Thanks..
